test/NuGet.Core.Tests/NuGet.Commands.Test/ProjectResolutionTests.cs

```diff
@@ -199,10 +199,9 @@ public class ProjectResolutionTests
                 Assert.Equal(0, result.GetAllUnresolved().Count);
                 Assert.NotNull(project2Lib);
                 Assert.Equal("Project project2 is not compatible with net45 (.NETFramework,Version=v4.5). Project project2 supports: net46 (.NETFramework,Version=v4.6)", issue.Format());
-                Assert.Equal(2, logger.ErrorMessages.Count());
+                Assert.Equal(1, logger.ErrorMessages.Count());
```

Comment:
The summary error message with the count was removed. It use to get the 1 error and then a 2nd summary message saying that there was 1 error of that type.